armi/utils/pathTools.py

```diff
@@ -116,25 +116,10 @@ def isAccessible(path):
         return False
 
 
-def getModAndClassFromPath(path):
-    """
-    Return the path to the module specified and the name of the class in the module.
-
-    Raises
-    ------
-    ValueError:
-        If the path does not exist or
-
-
-    """
-    pass
-
-
```

Comment:
This function is just egregious. How did this get in here??

Reply:
I can't tell. It had been there for three years.  

I know clean up PRs aren't sexy, but man, sometimes they are necessary.